src/sentry/api/bases/organization_events.py

```diff
@@ -156,6 +156,8 @@ def handle_results_with_meta(self, request, organization, project_ids, results):
             for key in data[0]:
                 if key not in meta:
                     meta[key] = "string"
+            # Remove any potential NaN or Inf
+            data = json.loads(json.dumps(data, ignore_nan=True))
```

Comment:
I think alternatively I could iterate through the results here instead, but this seemed much simpler

Reply:
The cost is no different than iteratively traversing the `data` object, and replacing NaN and Inf with None.

Reply:
Nah... this is much more expensive than iterating:
https://gist.github.com/wmak/ee299f095f4a3fa9ffacd3a76a6d56cf

Have updated to iterate instead and added an optional param if we decide to include more divides in the future.